### PR TITLE
integration of Bug3964 repairman

### PR DESCRIPTION
**Pull request by maxtaco**

### CODE REVIEW

File: go/engine/login_provisioned_device.go [hunk 1]
```diff
@@ -127,7 +127,14 @@ func (e *LoginProvisionedDevice) Run(ctx *Context) error {
 	}
 
 	// login was successful, unlock the device keys
-	return e.unlockDeviceKeys(ctx, me)
+	err = e.unlockDeviceKeys(ctx, me)
+	if err != nil {
+		return err
+	}
+
+	RunBug3964Repairman(e.G())
```
Comment (patrickxb) [inline go/engine/login_provisioned_device.go]: Just to be clear, this is going to run again in `sendNotification` when called via `Login` engine.  I see the config short-circuit, fine to run multiple times, just want to point it out.

**Comment by maxtaco** [comment]
@patrickxb this is worth another look. In particular, ddfe416 changes the way hourly background loops work.

**Comment by maxtaco** [comment]
The C/I failure isn't related, it's a KBFS flake.

**Comment by patrickxb** [comment]
ddfe416 looks good, continuing...

**Comment by maxtaco** [comment]
ok thanks. I merged back down to CORE-4002 and am going to stare it at all a bunch more before I merge.

**Pull request merged**